published/011-2018-sep-26.md

```diff
@@ -0,0 +1,62 @@
+
++ [IPFS is now available in 22 languages and counting!](https://www.transifex.com/ipfs/public/) Thanks to the recent IPFS localisation effort, IPFS is making its way around the globe.
++ [Cloudflare joined the InterPlanetary Mission.](https://blog.cloudflare.com/distributed-web-gateway/) Check out Cloudflare’s IPFS Gateway, announced during its first Crypto Week.
++ Watch the latest [Dynamic Data and Capabilities Working Group meeting video](https://www.youtube.com/watch?v=EiJE59uYNAk) from September 18 for a sneak peek at demos of identity manager and discussify, plus more.
++ The first Berlin IPFS Meetup is happening on October 11th! [Sign up to attend on Meetup.](https://www.meetup.com/IPFS-Berlin/events/254816369/)
++ The JavaScript implementation of IPFS just got a new homepage last month: [js.ipfs.io](https://js.ipfs.io/).
+
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
+
++ [Dweb: Creating Decentralized Organizations with Aragon](https://hacks.mozilla.org/2018/09/aragon-ethereum-dweb/), Mozilla Hacks, 19 Sep 2018 
++ [End-to-End Integrity with IPFS](https://blog.cloudflare.com/e2e-integrity/), Cloudflare, 17 Sep 2018 
++ [CloudFlare’s IPFS Gateway Makes it Easy to Create Distributed Web Sites](https://www.bleepingcomputer.com/news/technology/cloudflares-ipfs-gateway-makes-it-easy-to-create-distributed-web-sites/), 17 Sep 17 2018 
++ [An Interview with Textile](https://medium.com/textileio/an-interview-with-textile-6d52632f611b), 13 Sep 2018 
++ [The Centralized Secret](https://medium.com/@kyletut/the-centralized-secret-c7de795ddd9f), 12 Sep 2018 
++ [Decentralisation: the next big step for the world wide web](https://www.theguardian.com/technology/2018/sep/08/decentralisation-next-big-step-for-the-world-wide-web-dweb-data-internet-censorship-brewster-kahle), The Guardian, 8 Sep 2018, 
++ [Swapping bits and distributing hashes on the decentralized web](https://medium.com/textileio/swapping-bits-and-distributing-hashes-on-the-decentralized-web-5da98a3507), 4 Sep 2018, 
++ [Introducing Origin Messaging: Decentralized, Secure, and Auditable](https://medium.com/originprotocol/introducing-origin-messaging-decentralized-secure-and-auditable-13c16fe0f13e), Origin, 29 Aug 2018
++ [Dweb: Building Cooperation and Trust into the Web with IPFS](https://hacks.mozilla.org/2018/08/dweb-building-cooperation-and-trust-into-the-web-with-ipfs/), Mozilla Hacks, 29 Aug 2018
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [go-ipfs 0.4.17](https://github.com/ipfs/go-ipfs/releases/tag/v0.4.17) – IPFS implementation in Go
++ [ipfs-companion v2.5.0.11550 (Beta)](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.5.0.11550) – Browser extension that simplifies access to IPFS resources
```

Comment:
I would  de-emphasize beta release :)  This infor should be either removed, or merged with Stable one below.

Reply:
Is that because we don't want a lot of attention on companion yet or because this particular release isn't that big of a deal? If it's the latter I'd like to leave it called out since we haven't done a newsletter in years and it would be nice to highlight any project we want some attention on.

Reply:
It is okay to keep it (this beta ships with a nice welcome page), but it should be moved to be after item with Stable release and DNSLink and we need to update description of Beta release item to something like: _developers and enthusiasts can opt-in for Beta channel with hand-picked Dev Builds of our browser extension_